src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/appsettings.json

```diff
@@ -10,9 +10,9 @@
 ////#endif
   "Logging": {
       "LogLevel": {
-      "Default": "Information",

```

Comment:
Huh what changed here?

Reply:
Not sure - something weird in the merge, I guess. I can push a commit undoing this

Reply:
Replacing the file locally doesn't change anything in git, but I think this is a no-op anyways